pep-0683.rst

```diff
@@ -72,14 +72,19 @@ Here's a high-level look at the implementation:
 If an object's refcount matches a very specific value (defined below)
 then that object is treated as immortal.  The CPython C-API and runtime
 will not modify the refcount (or other runtime state) of an immortal
-object.
+object.  The runtime will now be explicitly responsible for deallocating
+all immortal objects during finalization, unless statically allocated.
+(See `Object Cleanup`_ below.)
```

Comment:
FYI, you might want to consider using [the `:ref:` role and ref target labels](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref) (e.g. `` :ref:`683-object-cleanup` linking to `` .. _683-object-cleanup:``) instead of relying on implicit internal links, as they don't break if you change the section headings, will raise warnings if they do break or are incorrect, and can be used across files, builders and even in other docs through Intersphinx.

Reply:
Thanks for pointing this out.  I'll take a look.